docs/content/filtering.md

```diff
@@ -5,31 +5,65 @@ description: "Filtering, includes and excludes"
 
 # Filtering, includes and excludes #
 
-Rclone has a sophisticated set of include and exclude rules. Some of
-these are based on patterns and some on other things like file size.
+Filter flags determine which files rclone`sync`, `move`, `ls`, `lsl`,
+`md5sum`, `sha1sum`, `size`, `delete` and `check` commands are applied
+to.
```

Comment:
This isn't a complete list of commands filters apply to - not sure I want to make a complete list so maybe a "such as" in  there?

Reply:
Good point, yes.